src/simplicial-hott/04-extension-types.rzk.md

```diff
@@ -558,16 +582,16 @@ In an extension type of a dependent type that is pointwise contractible, then we
   ( a : (t : ϕ ) → A t)
   ( is-contr-fiberwise-A : (t : ψ ) → is-contr ( A t))
   : (t : ϕ ) → (a t = first (is-contr-fiberwise-A t))
-  :=  \ t → 
+  :=  \ t →
```

Comment:
We should probably add something to `.vscode/settings.json` to automatically remove trailing whitespaces.

Reply:
I didn't even realize this was happening. My editor must be configured to delete trailing whitespace automatically...